openmc/__init__.py

```diff
@@ -16,6 +18,10 @@
 from openmc.executor import *
 from openmc.statepoint import *
 from openmc.summary import *
+from openmc.region import *
+from openmc.source import *
+from openmc.particle_restart import *
+from openmc.arithmetic import *
```

Comment:
Is `arithmetic` something we want imported for the user? I think this is more back-end code to keep hidden from the user.